numpy/core/tests/test_umath.py

```diff
@@ -2041,6 +2048,8 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
                     return NotImplemented
 
                 if method == 'at':
+                    if isinstance(inputs[0], A):
+                        inputs[0].info = info
```

Comment:
I was thinking about the `__array_ufunc__` example in the docs, which I thought was kept synced with one of the tests. Maybe it was not that one.

Yes, I think we should document that a tuple is allowed for `out` in those functions too.